featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
+	public static final int COLS = 10;
+	public static final int ROWS = 21;
+	
+	//currently 21 features. To be increased in the future
+	private final int numFeatures = 21;
+	private double[] scores = new double[numFeatures];
+	private int[][] grids;
+	
+	private int[] colHeights = new int[COLS];
+	
+	public featuresScore(State s) {
+		grids = s.getField();
+		
+		//First 10 are colHeight
+		for (int i = 0; i < COLS; i++) {
+			scores[i] = checkColHeight(i);
+		}
+		
+		//The next 9 are adjHeight
+		for (int i = 10; i < 10 + COLS - 1; i++) {
+			scores[i] = checkDiffAdjColHeights(i);
+		}
+		
+		//max col height
+		scores[20] = checkMaxColHeight();
+		
+		scores[21] = checkNumWallHoles();
+	}
+	
+	public double getScore(int index) {
+		return scores[index];
+	}
+	
+	public int getNumberOfFeatures() {
+		return numFeatures;
+	}
+	
+	//Check from above
+	public int checkColHeight(int col) {
+		int count = 0;
+		while (grids[count][col] != 0) {
+			count++;
+		}
+		return count;
+	}
+	
+	//Check absolute difference between adjacent column heights. This method compares grid[column] with grid[column+1]
```

Comment:
Check @vsharonlynn's PR for how to document Java methods. If you are using Eclipse, you can just type `/**` then type enter, it will auto-magically create a nicely-formatted Javadoc for the method, you just need to fill in the blanks.

And I prefer shorter lines, like ~80 characters max? (I feel I starting to sound like Damith alr)